rest-fights/src/main/java/io/quarkus/sample/superheroes/fight/service/FightService.java

```diff
@@ -210,4 +210,28 @@ Fight villainWonFight(Fighters fighters) {
 		fight.loserTeam = this.fightConfig.hero().teamName();
 		return fight;
 	}
+
+  @Timeout(value = 5, unit = ChronoUnit.SECONDS)
+  @Fallback(fallbackMethod = "fallbackHelloHeroes")
+  public Uni<String> helloHeroes() {
+    Log.debug("Invokes the Heroes microservice");
+    return this.heroClient.helloHeroes();
+  }
+
+  Uni<String> fallbackHelloHeroes() {
+    Log.warn("Could not invoke the Heroes microservice");
+    return Uni.createFrom().item("Could not invoke the Heroes microservice");
+  }
```

Comment:
This could also be

```java
return Uni.createFrom().item("Could not invoke the Heroes microservice")
  .invoke(Log::warn);
```

Reply:
Weird, I have the following exception now : 

```
java.lang.UnsupportedOperationException: Using io.quarkus.logging.Log is only possible with Quarkus bytecode transformation
```

Reply:
Weird, now I have:

```
[ERROR] Failures: 
[ERROR]   HeroClientTests.doesntRecoverFrom500:170 
Expecting actual throwable to be exactly an instance of:
  org.eclipse.microprofile.faulttolerance.exceptions.CircuitBreakerOpenException
but was:
  org.jboss.resteasy.reactive.ClientWebApplicationException: Received: 'Server Error, status code 500' when invoking: Rest Client method: 'io.quarkus.sample.superheroes.fight.client.HeroRestClient#findRandomHero'
 ```
 
 Gee, didn't know reactive programming was that difficult ;o)